### ISSUE TITLE
App navigation missing right border in high-contrast mode

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/nextcloud-vue#3351]
Also it feels a bit weird, but the "highlighted" entries seem less highlighted:

Normal | High-contrast
---|---
![Bildschirmfoto vom 2022-10-07 12-30-50](https://user-images.githubusercontent.com/213943/194533736-927c6ca6-1933-4694-9a4d-dbffc4928fe9.png) | ![Bildschirmfoto vom 2022-10-07 12-30-35](https://user-images.githubusercontent.com/213943/194533731-8ff1bd11-5e03-4732-84ce-2be4d4c934ed.png)
